services/checks/checks.go

```diff
@@ -93,16 +95,16 @@ func New(pmmVersion string) *Service {
 // Run runs checks service that grabs checks from Percona Checks service every interval until context is canceled.
 func (s *Service) Run(ctx context.Context) {
 	if f := os.Getenv(envCheckFile); f != "" {
-		s.l.Warnf("Use local test checks file: %s", f)
+		s.l.Warnf("Using local test checks file: %s", f)
 
-		data, err := ioutil.ReadFile(f)
+		data, err := ioutil.ReadFile(f) //nolint:gosec
 		if err != nil {
-			s.l.WithError(err).Error("Failed to read test checks file")
+			s.l.Errorf("Failed to read test checks file: %s.", err)
 			return
 		}
 		checks, err := check.Parse(bytes.NewReader(data))
 		if err != nil {
-			s.l.WithError(err).Error("Failed to parse test checks file")
+			s.l.Errorf("Failed to parse test checks file: %s.", err)
```

Comment:
@artemgavrilov in PMM, we don't use `WithError` and treat error messages as text, not as structured field.

Reply:
Ok, got it, will fix

Reply:
Already fixed that in this PR